sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.net8.0.cs

```diff
@@ -7258,6 +7397,40 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Compute.Models.PurchasePlan>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Compute.Models.PurchasePlan>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct RebalanceBehavior : System.IEquatable<Azure.ResourceManager.Compute.Models.RebalanceBehavior>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public RebalanceBehavior(string value) { throw null; }
+        public static Azure.ResourceManager.Compute.Models.RebalanceBehavior CreateBeforeDelete { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Compute.Models.RebalanceBehavior other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Compute.Models.RebalanceBehavior left, Azure.ResourceManager.Compute.Models.RebalanceBehavior right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Compute.Models.RebalanceBehavior (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Compute.Models.RebalanceBehavior left, Azure.ResourceManager.Compute.Models.RebalanceBehavior right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct RebalanceStrategy : System.IEquatable<Azure.ResourceManager.Compute.Models.RebalanceStrategy>
```

Comment:
this name is quite unclear, could we introduce a more detailed name?

Reply:
RebalanceStrategy: VmssRebalanceStrategy 
 added